projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/videopress-value-section.tsx

```diff
@@ -29,6 +29,10 @@ const VideoPressValueSection: FC< VideoPressValueSectionProps > = ( { isPluginAc
 	const currentViews = featuredStats?.data?.views?.current;
 	const currentWatchTime = featuredStats?.data?.watch_time?.current;
 
+	if ( currentViews === undefined || currentWatchTime === undefined ) {
```

Comment:
This is to handle the case where there was an error on the API call. 0's should be accepted if the site has videos